tests/Resources/ODS.robot

```diff
@@ -68,3 +71,36 @@ Usage Data Collection Should Not Be Enabled
     [Arguments]    ${msg}="Usage Data Collection" should not be enabled
     ${enabled}=    ODS.Is Usage Data Collection Enabled
     Should Not Be True    ${enabled}    msg=${msg}
+
+Uninstall RHODS From OSD Cluster
+    [Documentation]    Selects the cluster type and triggers the RHODS uninstallation
+    ${addon_installed}=     Is Rhods Addon Installed    ${CLUSTER_NAME}
```

Comment:
I suggest you pass ${CLUSTER_NAME} as parameter instead of using a global variable

Reply:
needed for the suite teardown

Reply:
yup, you're right. It was necessary for a previous version of the PR